src/System Application/App/Email/src/Account/EmailAccounts.Page.al

```diff
@@ -400,6 +402,21 @@ page 8887 "Email Accounts"
         CurrPage.Update(false);
     end;
 
+    local procedure FilterToConnectorv2Accounts(var EmailAccounts: Record "Email Account")
+    var
+        IConnector: Interface "Email Connector";
+    begin
+        if EmailAccounts.IsEmpty() then
+            exit;
+
+        repeat
+            IConnector := EmailAccounts.Connector;
+            if not (IConnector is "Email Connector v2") then
+                EmailAccounts.Delete();
+        until EmailAccounts.Next() = 0;
+        if EmailAccounts.FindSet() then;
```

Comment:
Was showing all accounts prior, as such this was added. Works without now. Removing.
```suggestion
```